src/viam/sdk/components/camera/camera.hpp

```diff
@@ -4,6 +4,7 @@
 #pragma once
 
 #include <bitset>
+#include <boost/endian/conversion.hpp>
```

Comment:
can we put this `include` in the `.cpp`? Seems like an implementation detail that users don't need to know about.

Also note that the `include`s are sorted into different groups that might be a bit unintuitive. This should go with the `google/protobuf` includes in the `.cpp` as a third party include.